lbry/lbry/schema/base.py

```diff
@@ -119,3 +119,6 @@ def __getitem__(self, item) -> I:
 
     def __delitem__(self, key):
         del self._message[key]
+
+    def __eq__(self, values: List[str]) -> bool:
+        return self._message == values
```

Comment:
Where is the equality method used?

Reply:
Only in tests I think, to compare tags with lists of strings more simply.